easybuild/easyconfigs/t/TrimGalore/TrimGalore-0.4.5-GCCcore-7.3.0-Python-3.7.4-bare.eb

```diff
@@ -0,0 +1,38 @@
+# 0.4.2 modified by:
+# Adam Huffman
+# The Francis Crick Institute
+# TODO: deal with the hardcoding of /usr/bin/perl
+
+easyblock = 'Tarball'
+
+name = 'TrimGalore'
+version = '0.4.5'
+versionsuffix = '-Python-%(pyver)s-bare'
+
+homepage = 'http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/'
+description = """A wrapper tool around Cutadapt and FastQC to consistently apply 
+ quality and adapter trimming to FastQ files, with some extra functionality for 
+ MspI-digested RRBS-type (Reduced Representation Bisufite-Seq) libraries."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+sources = ['%(version)s.zip']
+source_urls = ['https://github.com/FelixKrueger/%(name)s/archive/']
+checksums = ['c49cd40629d399bd24320ac2f934deee7be758f58465d8c63da799f767f895d0']
+
+#install_cmd = "cp -a * %(installdir)s && chmod +x %(installdir)s/%(namelower)s"
+
+dependencies = [
+    ('FastQC', '0.11.8', '-Java-11-LTS', ('dummy', '')),
+    ('cutadapt', '2.6', '-Python-3.7.4-bare'),
+    ('Python', '3.7.4', '-bare'),
+]
+
+sanity_check_paths = {
+    'files': ["trim_galore"],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```

Comment:
Die eerste 'dirs' is er niet; normaal zou de sanity check daar over struikelen, maar eronder staat het nog een keer en dan checkt hij op een lege set mappen. Beter maken we er dit van:
```
'dirs': ['Docs', 'test_files'],
```

Reply:
heb ik aangepast!